src/plugins/unified_histogram/public/layout/layout.tsx

```diff
@@ -14,7 +14,7 @@ import useObservable from 'react-use/lib/useObservable';
 import { createHtmlPortalNode, InPortal, OutPortal } from 'react-reverse-portal';
 import { css } from '@emotion/css';
 import type { Datatable, DatatableColumn } from '@kbn/expressions-plugin/common';
-import type { DataView, DataViewField } from '@kbn/data-views-plugin/public';
+import { type DataView, DataViewField } from '@kbn/data-views-plugin/public';
```

Comment:
also `DataViewField` is just used as type

Reply:
Nice catch https://github.com/elastic/kibana/pull/193820/commits/1b95c94f109e37bb33713962f18fa3a3dd8a16af